src/test/java/net/mcreator/integration/generator/GTCommandArgBlocks.java

```diff
@@ -101,7 +101,7 @@ public static void runTest(Logger LOG, String generatorName, Random random, Work
 									additionalXML.append("<field name=\"").append(field).append("\">1.23d</field>");
 									processed++;
 									break;
-								case "field_input":
+								case "field_javaname":

```

Comment:
You should add this as an additional case for field_javaname, keep field_input in case some other block adds it

Reply:
Like in https://github.com/MCreator/MCreator/commit/89c06599c6881eef06d0efef6da3ddbc1f442e28

Reply:
I was not sure if it would be useless or for this PR.

Reply:
Just to future proof it :)